src/components/Metropolis/components/Header.tsx

```diff
@@ -0,0 +1,159 @@
+import { cn } from '@site/src/utils';
+import React, { useEffect, useState } from 'react';
+import { slide as Menu } from 'react-burger-menu';
+import BurgerIcon from './icons/BurgerIcon';
+import CrossIcon from './icons/CrossIcon';
+import { DOCS_URL } from '../constants';
+
+const LINKS = [
+    {
+        label: 'Docs',
+        path: DOCS_URL,
+        style: 'link',
```

Comment:
dont think we're quite used to the pattern of defining `link` and `button` 
let me think of a better way